Introduction.md

```diff
@@ -32,6 +32,22 @@ There are many different reasons for companies to contribute to existing OSS pro
 
 ## Maturity levels
 
-TODO: Describe maturity model of engaging with open source, from passive use, over contributions to existing projects, starting own project, to strategic cocreation across company boundaries. See [issue #8](https://github.com/Open-Source-Compliance/outbound-oss/issues/8).
+Corporate adoption of open source can typically be described using a model of maturity levels. These levels characterize how, with growing maturity, open source is used in an increasingly effective fashion to drive value and realize business needs. How outbound open source is handled is a distinguishing factor between the different levels of maturity. A critical factor is the insight that influencing the open source ecosystem is mainly done by participation and doing work as contributions to open source projects.
```

Comment:
"... doing work as contributions to open source projects.": Better (IMO): "... contribution to open source projects."

Reply:
Good point. Will change it.